indexer/packages/base/src/constants.ts

```diff
@@ -3,3 +3,12 @@
 export const STATS_NO_SAMPLING: number = 1;
 
 export const STATS_FUNCTION_NAME: string = 'generic_function';
+
+export const ONE_SECOND_IN_MILLISECONDS: number = 1000;
```

Comment:
This seems a lot more verbose than just a number + comment.

Reply:
Personally I think it's cleaner to have the code speak for itself rather than have comments explain why this random constant is what it is